var/www/skitter.fanfic.dev/index.html

```diff
@@ -1,88 +1,111 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
re: this change and some others, did you run an autoformatter on this? Which did you use and what settings?

Reply:
For this specific change, I was just being picky. Although every time I've seen a HTML5 doctype declaration it was always in uppercase so I guess it's the standard convention.

I used [Emmet Abbreviation](https://code.visualstudio.com/docs/editor/emmet). It is built into [visual studio code](https://code.visualstudio.com/). I did not need to change the default settings either.
Emmet produces this:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Document</title>
</head>
<body>
	|
</body>
</html>
```

Reply:
It's case insensitive, but looking at MDN and some examples it does seem to usually be capitalized so I'm fine with changing it, was hoping there was a linter that enforced subjective style things like that.

That emmet plugin seems to be something to save typing and doesn't really discuss auto-formatting. Am I misunderstanding?

Reply:
> ... was hoping there was a linter that enforced subjective style things like that.

Not that I know of

>  That emmet plugin seems to be something to save typing and doesn't really discuss auto-formatting. Am I misunderstanding?
You're not. 
https://docs.emmet.io/actions/

Reply:
Alright; I'll worry about autoformatting later then.